src/components/maps/Map.js

```diff
@@ -86,14 +86,30 @@ const mapDispatchToPropsFactory = () => {
 				onClick: () => {
 					dispatch(Action.maps.setMapSetActiveMapKey(ownProps.stateMapKey));
 				},
-				onLayerClick: (mapKey, layerKey, selectedFeatureKeys) => {
+				onLayerClick: (
+					mapKey,
+					layerKey,
+					selectedFeatureKeys,
+					position,
+					options
+				) => {
 					dispatch(
```

Comment:
Shouldn't the dispatch also be conditional? Do we always want both dispatch and bubbling above?

Reply:
I did not want to change API, but this handler onLayerClick is in "controlled" section, so I think it should dispatch onLayerClick every time.

Reply:
The dispatch is Action.maps.setLayerSelectedFeatureKeys - should that be always done even when there is another onLayerClick method supplied?

Reply:
That's better.  Hmm, I agree.